cl/citations/tasks.py

```diff
@@ -237,7 +253,46 @@ def store_opinion_citations_and_update_parentheticals(
         opinion.save(index=False)
 
 
-def store_recap_citations_and_update_parentheticals(
-    document: RECAPDocument, index: bool
-):
-    pass
+def store_recap_citations(document: RECAPDocument) -> None:
+    """
+    Identify citations from federal filings to opinions.
+
+    :param document: A search.RECAPDocument object.
+
+    :return: None
+    """
+
+    document_text = document.plain_text
```

Comment:
We should run `clean_text(text, ["all_whitespace"])` on this before looking for citations, right?